src/error.rs

```diff
@@ -37,12 +37,12 @@ impl Error {
 #[derive(Error, Diagnostic, Debug)]
 pub enum ErrorKind {
     /// this error is for testing. You can use it when you want to quickly see in what file and what line of code the error is.
-    #[error(
-        "Unexpected error: {0}. Please report this error on https://github.com/mimoo/noname/issues"
-    )]
+    #[error("An unexpected error occured: {0}")]
```

Comment:
oh the text was good! We should still ask people to report the error, I meant the comment above